types/ffmpeg.js/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
+        "npm-naming": false
```

Comment:
please don't disable this rule for new packages. If it a non-npm package the header should read `// Type definitions for non-npm package ffmpeg.js 3.1`